packages/components/src/markdown/index.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router';
+import omit from 'lodash/omit';
+import clsx from 'clsx';
+import marked from 'marked';
+import { invoke } from 'nebenan-helpers/lib/utils';
+import { stripOriginFromUrl } from 'nebenan-helpers/lib/routes';
+import { historyPropTypes } from 'nebenan-react-hocs/lib/history';
+import styles from './index.module.scss';
```

Comment:
as usual: we will need to be careful migrating to this component. it may cause styling to break

Reply:
Very good point. Since we don't migrate everything to Markdown now - we should be good but this will definitely break css selectors.

To migrate nebenan-components Markdown usages, we would need to add theme support - should we do it now or later?

Reply:
I think we can get away with root classname here